Mathlib/NumberTheory/NumberField/CanonicalEmbedding/FundamentalCone.lean

```diff
@@ -262,8 +271,10 @@ theorem integralPoint_ne_zero (a : integralPoint K) :  (a : mixedSpace K) ≠ 0
 
 open scoped nonZeroDivisors
 
-/-- For `a : integralPoint K`, the unique nonzero algebraic integer whose image by
-`mixedEmbedding` is equal to `a`, see `mixedEmbedding_preimageOfIntegralPoint`. -/
+/-- For `a : fundamentalCone K`, the unique nonzero algebraic integer `x` which image by
+`mixedEmbedding` is equal to `a`. Note that we state the fact that `x ≠ 0` by saying that `x` is
+a nonzero divisors since we will use later on the isomorphism
+`Ideal.associatesNonZeroDivisorsEquivIsPrincipal`, see `integralPointEquiv`. -/
```

Comment:
```suggestion
/-- For `a : fundamentalCone K`, the unique nonzero algebraic integer `x` such that its image by
`mixedEmbedding` is equal to `a`. Note that we state the fact that `x ≠ 0` by saying that `x` is
a nonzero divisors since we will use later on the isomorphism
`Ideal.associatesNonZeroDivisorsEquivIsPrincipal`, see `integralPointEquiv`. -/
```
I am not 100% this is better in English, feel free to find a native speaker :)